core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -79,8 +79,8 @@ oppia.factory('UrlInterpolationService', [
         var INVALID_VARIABLE_REGEX = /(<{2,})(\w*)(>{2,})/;
 
         // Parameter values can only contain alphanumerical characters, spaces,
-        // hyphens, underscores, or periods.
-        var VALID_URL_PARAMETER_VALUE_REGEX = /^(\w| |_|-|[.])+$/;
+        // hyphens, underscores, periods or the equal to symbol.
+        var VALID_URL_PARAMETER_VALUE_REGEX = /^(\w| |_|-|[.]|=)+$/;
```

Comment:
I don't know. That was my question... if we want to do this properly, we need to actually understand what's going on.